src/DotNetWorker.Core/StartupHook.cs

```diff
@@ -48,9 +42,19 @@ static bool WaitOnDebugger(int cycle)
             return true;
         }
 
-        for (int i = 0; WaitOnDebugger(i); i++)
+        if (string.Equals(jsonOutputEnabled, bool.TrueString, StringComparison.OrdinalIgnoreCase))
+        {
+            Console.WriteLine($"azfuncjsonlog:{{ \"name\":\"dotnet-worker-startup\", \"workerProcessId\" : { Environment.ProcessId } }}");
+        }
+
+        if (string.Equals(debuggerWaitEnabled, bool.TrueString, StringComparison.OrdinalIgnoreCase))
         {
-            Thread.Sleep(SleepTime);
+            Console.WriteLine($"Azure Functions .NET Worker (PID: { Environment.ProcessId }) initialized in debug mode. Waiting for debugger to attach...nblswdivnsldvnsldiv");
```

Comment:
`attach...nblswdivnsldvnsldiv` I am guessing you didn't mean to type that?

Reply:
It was just a test to make sure you were really paying attention when reviewing. 😆 

Sorry, fixing :)

Reply:
Someone's sleeping on the job 😂